yarn.lock

```diff
@@ -13707,6 +13702,17 @@ fast-glob@3.2.5, fast-glob@^3.0.3, fast-glob@^3.1.1, fast-glob@^3.2.2, fast-glob
     micromatch "^4.0.2"
     picomatch "^2.2.1"
 
+fast-glob@^3.2.5:
```

Comment:
All the `fast-glob` dependencies listed above that currently resolves to version 3.2.5 could be consolidated into this one (this requires that we bump the `globby/fast-glob` resolution in `package.json` to `3.2.7` as well, which can also safely be done).

Reply:
Done in 417d703331306493dcc17d6aa80fbec670ecc3c1